src/sorcha/modules/PPLinkingFilter.py

```diff
@@ -88,7 +88,7 @@ def PPLinkingFilter(
         min_window_nights=min_tracklets,  # minimum size of a window as the rolling window is moved
         by_object=True,  # split observations by object instead of by window [set to True if you want ignore_after_discovery=True]
         discovery_probability=detection_efficiency,  # probability of discovery for any discovery chance
-        num_jobs=30,  # number of parallel jobs
+        num_jobs=1,  # number of parallel jobs
```

Comment:
Note that I set this to 1. @maxwest-uw noted that there seemed to be quite a lot of overhead associated with values other than 1, so to avoid that, and until we can characterize that penalty more clearly, I suggest that we keep this as 1 for now.